.circleci/config.yml

```diff
@@ -3,28 +3,38 @@
 
 version: 2.1
 
-env: &env
-  TERM: dumb
-  CI: "true"
-  USER: circleci
-
-jobs:
-  build:
+executors:
+  chart-releaser:
     docker:
-    - image: cimg/base:2021.04
+       - image: quay.io/helmpack/chart-releaser:v1.5.0
     environment:
-      <<: *env
+      LC_ALL: "C.UTF-8"
+    resource_class: medium
+
+commands:
+  build-cr:
     steps:
-      - checkout
       - run:
-         name: build
-         command: |
-           set -euo pipefail
+          command: apk add helm yq github-cli git jq openssh bash
```

Comment:
Can we add a nix setup here to avoid running the apk add every time?

Reply:
I tried this out here: https://app.circleci.com/pipelines/github/adamma-da/test-chart-releaser/192/workflows/96ebd360-39c1-4581-8606-f834a4fdc5b8/jobs/156
It seems to work without any issues although it makes the build time quite long. To keep this MR clean I would prepare another one as we agreed

Reply:
We need to commit a Docker image, same issue as in our other pipelines... Nix will install packages too 😞